### PR TITLE
doc: fix typos

### PR DESCRIPTION
**Pull request by botengyao**
Signed-off-by: Boteng Yao <boteng@google.com>

Commit Message:
Additional Description:
Risk Level: L
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: api/envoy/service/auth/v2/attribute_context.proto [hunk 1]
```diff
@@ -67,9 +67,10 @@ message AttributeContext {
     // `URI Subject Alternative Names`, `DNS Subject Alternate Names` or `Subject` in that order.
     // The primary identity should be the principal. The principal format is issuer specific.
     //
-    // Example:
-    // *    SPIFFE format is `spiffe://trust-domain/path`
-    // *    Google account format is `https://accounts.google.com/{userid}`
+    // Examples:
+    //
+    // - SPIFFE format is `spiffe://trust-domain/path`.
```
Comment (phlax) [inline api/envoy/service/auth/v2/attribute_context.proto:72]: not sure the spiffe: link should be a literal - but if so it should use double backticks ` `` `

File: api/envoy/service/auth/v2/attribute_context.proto [hunk 2]
```diff
@@ -108,7 +109,7 @@ message AttributeContext {
     // the URL path and query-string. No decoding is performed.
     string path = 4;
 
-    // The HTTP request `Host` or 'Authority` header value.
+    // The HTTP request `Host` or `Authority` header value.
```
Comment (phlax) [inline api/envoy/service/auth/v2/attribute_context.proto:112]: in this case i think it definitely should be a literal - so could you use double backticks please

File: api/envoy/service/auth/v2/attribute_context.proto (hunk 1 above)
Comment (botengyao) [reply api/envoy/service/auth/v2/attribute_context.proto]: I see, we should use double backticks for v3.

Comment (phlax) [reply api/envoy/service/auth/v2/attribute_context.proto]: its more just how it is rendered from rst - we previously had a policy doc that said use single backticks - but we changed that a long time ago

better leave the v2 protos alone tho - yep

File: api/envoy/service/auth/v3/attribute_context.proto [hunk 3]
```diff
@@ -121,7 +122,7 @@ message AttributeContext {
     // the URL path and query-string. No decoding is performed.
     string path = 4;
 
-    // The HTTP request ``Host`` or '`Authority`` header value.
+    // The HTTP request ``Host`` or ``Authority`` header value.
```
Comment (lizan) [inline api/envoy/service/auth/v3/attribute_context.proto:125]:
```suggestion
    // The HTTP request ``Host`` or ``:authority`` header value.
```

Comment (botengyao) [reply api/envoy/service/auth/v3/attribute_context.proto]: Thanks. Done.

**Comment by alyssawilk** [comment]
cc @lizan ping?

**Comment by alyssawilk** [comment]
@lizan can you take a look a this today please?

**Review by adisuissa — APPROVED** [review]
/lgtm api
LGTM, thanks!

**Comment by botengyao** [comment]
I think this can be merged? Thanks! cc @phlax.

**Review by phlax — APPROVED** [review]
lgtm, thanks @botengyao

**Pull request merged**